docs/conversion/sakai_22_mysql_conversion.sql

```diff
@@ -30,3 +30,6 @@ ALTER TABLE GB_GRADEBOOK_T ADD COLUMN COMPARING_DISPLAY_ALLITEMS BIT(1) DEFAULT
 ALTER TABLE GB_GRADEBOOK_T ADD COLUMN COMPARING_RANDOMIZEDATA BIT(1) DEFAULT FALSE NOT NULL;
 -- End SAK-44967
 
+-- SAK-46022
+ALTER TABLE COMMONS_POST ADD COLUMN PRIORITY BIT(1);
```

Comment:
Can you add the same default and non-null stipulation as you did in the Oracle script to the MySQL script?

Reply:
Got it. Thank you for catching that!